Sprint-1/errors/4.js

```diff
@@ -1,2 +1,2 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const twelveHourClockTime = "20:53";
+const twentyFourHourClockTime = "08:53";
```

Comment:
(Extra question) Have you also noticed the variable names do not quite match the values assigned to the variable?

Reply:
Yes, I noticed that the variable names did not match the values. But was confused as to the reason why that was so.